hooks/emitter/decoder.go

```diff
@@ -661,6 +670,39 @@ func DecodeDescription(des stakingtypes.Description) common.JsDict {
 	}
 }
 
+func DecodeFeedsMsgSubmitSignalPrices(msg *feedstypes.MsgSubmitSignalPrices, detail common.JsDict) {
+	detail["validator"] = msg.GetValidator()
+	detail["timestamp"] = msg.GetTimestamp()
+	detail["signal_prices"] = msg.GetSignalPrices()
+}
+
+func DecodeFeedsMsgVote(msg *feedstypes.MsgVote, detail common.JsDict) {
+	detail["voter"] = msg.GetVoter()
+	detail["signals"] = msg.GetSignals()
+}
+
+func DecodeFeedsMsgUpdateReferenceSourceConfig(msg *feedstypes.MsgUpdateReferenceSourceConfig, detail common.JsDict) {
+	rsc := msg.GetReferenceSourceConfig()
+	detail["registry_ipfs_hash"] = rsc.RegistryIPFSHash
+	detail["registry_version"] = rsc.RegistryVersion
```

Comment:
this as well. we should make it similar to the message as much as possible.

Reply:
ok